docs/usage/managed_seed.md

```diff
@@ -71,4 +71,32 @@ spec:
 
 Note the `seed.gardener.cloud/gardenlet: local` label above. It stands for the label that is used in a `seedSelector` field of a `gardenlet` that takes care of multiple seeds. This label can be omitted if the `seedSelector` field selects all seeds. If there is no gardenlet running outside the cluster and selecting the seed, it won't be reconciled and no shoots will be scheduled on it.
 
-For an example that uses non-default configuration, see [55-managed-seed-seedtemplate.yaml](../../example/55-managedseed-seedtemplate.yaml)
\ No newline at end of file
+For an example that uses non-default configuration, see [55-managed-seed-seedtemplate.yaml](../../example/55-managedseed-seedtemplate.yaml)
+
+## Migrating from the `use-as-seed` Annotation to `ManagedSeeds`
```

Comment:
Maybe you want to explicitly link this section in https://github.com/gardener/gardener/blob/master/docs/usage/shooted_seed.md?
Something like
```
If you want to migrate existing Shoots with the `use-as-seed` annotation, you can find instructions [here](...).
```

Reply:
Done